chapter_6/Readme.md

```diff
@@ -0,0 +1,48 @@
+# Chapter 6: Strings and Text
+
+## What you should see:
+```
+(113796143_chapter_6_master) Kyle Skinner
+Kyle:chapter_6 $ ./ex6.rb
+There are 10 types of people.
+Those who know binary and those who don't.
+I said: There are 10 types of people..
+I also said: 'Those who know binary and those who don't.'.
+Isn't that joke so funny?! false
+This is the left side of...a string with a right side.
+```
+
+
+## Study Drills:
+>Go through this program and write a comment above each line explaining it.
+
+See ex6.rb
+
+>Find all the places where a string is put inside a string. There are four places.
+
+See ex6.rb lines: 6, 12, 20, 22, 27.
+
+>Are you sure there are only four places? How do you know? Maybe I like lying.
+
+There are 6 instances of interpolation
```

Comment:
You only list 5 places above...

Also, lines 6 and 27 aren't strings within strings. Why is that? What is being interpolated into a string?